source/jsonnet/northern-ireland/individual/blocks/identity-and-health/no_religion_other.jsonnet

```diff
@@ -4,13 +4,6 @@ local rules = import 'rules.libsonnet';
 local question(title) = {
   id: 'no-religion-other-question',
   title: title,
-  guidance: {
-    contents: [
-      {
-        description: 'This question is <strong>voluntary</strong>',
-      },
-    ],
-  },
```

Comment:
The sheet has no mention of `no_religion`, is this what Donna meant by `childhood-religion-other`?

Reply:
There is some cross over from Petes story here, as he has changed no_religion_other to childhood-religion-other, however thats was after this PR so someone has been playing with time travel